src/sources/invictus.ts

```diff
@@ -26,6 +29,9 @@ export class StakedInvictusMarketSource implements MarketSource {
    */
   constructor(connection: Connection) {
     this.connection = connection;
+    this.provider = new SolanaReadonlyProvider(
+      connection
+    ) as unknown as Provider;
```

Comment:
Yep, will have to address this in `saber-common`. They really should be accepting `ReadonlyProvider` in `token-utils` always